sample/src/main/res/layout/activity_main.xml

```diff
@@ -1,23 +1,17 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity"
-    android:orientation="vertical"
-    android:background="#fff">
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent" android:layout_height="match_parent" android:background="#fff"
+    android:orientation="vertical" tools:context=".MainActivity">
```

Comment:
It's just the style we have in other projects. Can I set it to be the standard one just in this project? I don't want to set a global rule...

Reply:
Let's push like this, I could do this later ;)

Reply:
I actually prefer the way it is now, because it's easier to see view hierarchy even in XML. Many times, when a view has a lot of attributes, it causes me to scroll a lot and try to remember what's happening around. Hard to read and understand what's going on, while also seeing the attributes. 

But I do understand the request: it's the standard. In the past I also wanted to use WhiteSmith code style for Java, but I left it in the end because I couldn't convince anyone to use it (I still think it's better than all the rest).